MimeKit/Utils/MimeUtils.cs

```diff
@@ -86,10 +86,16 @@ public static string GenerateMessageId (string domain)
 
 			ulong value = (ulong) DateTime.UtcNow.Ticks;
 			var id = new ValueStringBuilder (64);
+
+#if NET6_0_OR_GREATER
+			Span<byte> block = stackalloc byte[8];
+
```

Comment:
Can we stackalloc in both cases?

Reply:
sadly, no RandomNumberGenerator overload accepting spans on older runtimes.

Reply:
No, I mean, instead calling `new byte[8]`, can we do `stackalloc byte[8]` in the non-6_0_OR_GREATER case?

Reply:
Looks like the subsequent call is hidden from the PR. byte[] block is passed into GetRandomBytes(byte[]). This method cannot be spanified, so we can't stackalloc block. 


```csharp
var block = new byte[8];

GetRandomBytes (block);
```